packages/ui/src/components/Transactions/TransactionList.tsx

```diff
@@ -160,11 +160,11 @@ const TransactionList = ({ className }: Props) => {
         const filtered = res.filter((agg) => agg !== undefined) as AggregatedData[]
         const timestampObj: AggGroupedByDate = {}
 
-        // sort by date
+        // sort by date, the newest first
         const sorted = filtered.sort((a, b) => {
           if (!a.timestamp || !b.timestamp) return 0
```

Comment:
As a suggestion, I would move it to utils

Reply:
Thank. While it's too specific for a shared util (it's dealing with `AggregatedData` which is defined here), I've defined this specific function outside of the component.